frontend/src/actions/paginationActions.js

```diff
@@ -11,6 +11,12 @@ import {
   setCPTPageOptions,
   sliceCPTTableRows,
 } from "./homeActions";
+import {
+  fetchOLSJobsData,
+  setOLSOffset,
+  setOLSPage,
+  setOLSPageOptions,
+} from "./olsActions";
```

Comment:
This was duplicated ... probably because at some point this import was shifted down after Telco.